e2e/helpers/puppeter_helper.js

```diff
@@ -43,8 +43,8 @@ module.exports = class PuppeteerHelpers extends Helper {
     }
   }
 
-  async waitForAnySelector(selectors) {
-    return this.waitForSelector([].concat(selectors).join(','));
+  async waitForAnySelector(selectors, maxWaitInSecond) {
```

Comment:
one minor thing to note about this and the base method that it uses, the only selectors that it properly works on is css selectors (see [here](https://github.com/puppeteer/puppeteer/blob/v5.5.0/docs/api.md#pagewaitforselectorselector-options) which links to [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors))